### PR TITLE
Social Media Handles Added In Footer

### PR DESCRIPTION
**Pull request by 2580ayush2580**
I have added social media links in footer.
I have added social media icons from https://fontawesome.com/.

### CODE REVIEW

File: team/index.html [hunk 1]
```diff
@@ -572,6 +581,7 @@ <h2 class="avenirDemi">ncs@jssaten.ac.in</h2>
 	</section>
 	</div>
 	<!-- 	New page 2019 -->
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/js/all.min.js"></script>
```
Comment (rajat2502) [inline team/index.html]: Don't use the whole library for just 5 icons. You should use SVGs. Search [this](https://icons8.com/icons/) website you will find all the svgs here.

**Review by rajat2502 — CHANGES_REQUESTED** [review]

**Comment by rajat2502** [comment]
Use SVGs in all the files and it would be GTG.

File: team/index.html (hunk 1 above)
Comment (2580ayush2580) [reply team/index.html]: ok sir

**Comment by 2580ayush2580** [comment]
I  have added svg files of icons

File: index.html [hunk 2]
```diff
@@ -195,12 +195,21 @@ <h2 class="avenirDemi">ncs@jssaten.ac.in</h2>
...
-                  <img src="img/facebook.svg" class="pr-3" height="36px" />
-                </a>
-                <a href="http://github.com/ncs-jss" target="_blank" style="text-decoration: none;">
-                  <img src="img/github-icon.svg" height="36px" />
-                </a>
+                <a href="http://facebook.com/nibblecomputersociety" target="_blank" style="text-decoration: none;margin-right: 5px;">
+                            <img src="img/facebook.svg" height="28px" />
+                        </a>
+                  <a href="https://www.instagram.com/hackncs/" target="_blank" style="text-decoration: none;margin-right: 5px;">
+                           <img src="img/instagram%20.svg" height="28px" />
```
Comment (rajat2502) [inline index.html]: format the code in all the files.

**Review by rajat2502 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (2580ayush2580) [reply index.html]: done sir  i have formatted the code.

**Comment by rajat2502** [comment]
Looks fine for now.
Try not to use inline-CSS in future commits.

**Review by rajat2502 — APPROVED** [review]

**Comment by 2580ayush2580** [comment]
ok sir i will remember.

**Pull request merged**